ui/components/multichain/edit-accounts-modal/edit-accounts-modal.tsx

```diff
@@ -90,144 +88,144 @@ export const EditAccountsModal: React.FC<EditAccountsModalProps> = ({
   const checked = allAreSelected();
   const isIndeterminate = !checked && selectedAccountAddresses.length > 0;
 
-  const hostName = getURLHost(activeTabOrigin);
-
   const defaultSet = new Set(defaultSelectedAccountAddresses);
   const selectedSet = new Set(selectedAccountAddresses);
 
   return (
-    <>
-      <Modal
-        isOpen
-        onClose={onClose}
-        data-testid="edit-accounts-modal"
-        className="edit-accounts-modal"
-      >
-        <ModalOverlay />
-        <ModalContent>
-          <ModalHeader onClose={onClose}>{t('editAccounts')}</ModalHeader>
-          <ModalBody paddingLeft={0} paddingRight={0}>
-            {showAddNewAccounts ? (
-              <Box paddingLeft={4} paddingRight={4} paddingBottom={4}>
-                <CreateEthAccount
-                  onActionComplete={() => setShowAddNewAccounts(false)}
+    <Modal
+      isOpen
+      onClose={onClose}
+      data-testid="edit-accounts-modal"
+      className="edit-accounts-modal"
+    >
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader onClose={onClose}>{t('editAccounts')}</ModalHeader>
+        <ModalBody
+          paddingLeft={0}
+          paddingRight={0}
+          className="edit-accounts-modal__body"
+        >
+          {showAddNewAccounts ? (
+            <Box paddingLeft={4} paddingRight={4} paddingBottom={4}>
+              <CreateEthAccount
+                onActionComplete={() => setShowAddNewAccounts(false)}
+              />
+            </Box>
+          ) : (
+            <>
+              <Box
+                padding={4}
+                display={Display.Flex}
+                justifyContent={JustifyContent.spaceBetween}
+              >
+                <Checkbox
+                  label={t('selectAll')}
```

Comment:
should we also change the label based on the same `onClick` logic?

Reply:
I don't think so @andreahaku , since the empty state of the checkbox refers to All unselected, the intermediate to some selected and the check would be that all are selected

<img width="117" alt="Screenshot 2024-10-16 at 11 30 28" src="https://github.com/user-attachments/assets/2545a1ca-e584-4755-950d-bd462094dabd">

This pattern is common, without altering the text